relay-server/src/actors/store.rs

```diff
@@ -173,23 +301,35 @@ impl StoreForwarder {
                 .as_deref()
                 .map(make_distinct_id)
                 .unwrap_or_default(),
+            quantity: 1,
             seq: if session.init { 0 } else { session.sequence },
             received: protocol::datetime_to_timestamp(session.timestamp),
             started: protocol::datetime_to_timestamp(session.started),
             duration: session.duration,
-            status: session.status,
+            status,
             errors: session
                 .errors
                 .min(u16::max_value().into())
                 .max((session.status == SessionStatus::Crashed) as _) as _,
             release: session.attributes.release,
             environment: session.attributes.environment,
+            user_agent: session.attributes.user_agent,
+            os: session.attributes.os,
+            runtime: session.attributes.runtime,
             sdk: client.map(str::to_owned),
             retention_days: event_retention,
-        });
+        })
```

Comment:
Looks like an unintentional change.

```suggestion
        });
```

Reply:
this was intentional, as that function returns a `Result` now